inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -257,20 +268,35 @@ def raise_if_match_wf_in_error_or_initial(obj, eng):
         None
 
     """
-    def _filter(base_record, match_result):
-        return get_value(
-            match_result, '_source._workflow.status'
-        ) in ('ERROR', 'INITIAL')
-
-    matched_ids = pending_in_holding_pen(obj, _filter)
+    matched_ids = get_wfs_not_completed(obj)
     if bool(matched_ids):
```

Comment:
what''s the `bool` for?

Reply:
I have no idea. It's not my code ;)
In this situation list would be converted to bool so it looks like forcing cast to bool is unnecessary.